cms/templates/certificates.html

```diff
@@ -30,8 +32,16 @@
 
 <%block name="requirejs">
   require(["js/certificates/factories/certificates_page_factory"], function(CertificatesPageFactory) {
-      if(${escape_json_dumps(has_certificate_modes)}) {
-          CertificatesPageFactory(${escape_json_dumps(certificates) | n}, "${certificate_url}", "${course_outline_url}", ${escape_json_dumps(course_modes) | n}, ${escape_json_dumps(certificate_web_view_url) | n}, ${escape_json_dumps(is_active) | n}, ${escape_json_dumps(certificate_activation_handler_url) | n} );
+      if(${has_certificate_modes | n, dump_js_escaped_json}) {
+          CertificatesPageFactory(
+              ${certificates | n, dump_js_escaped_json},
+              "${certificate_url | n, js_escaped_string}",
+              "${course_outline_url | n, js_escaped_string}",
+              ${course_modes | n, dump_js_escaped_json},
+              "${certificate_web_view_url | n, js_escaped_string}",
```

Comment:
Note that once we make a decision, I will update and manually test to be sure I don't introduce issues.